src/03-control-flow.js

```diff
@@ -14,7 +14,13 @@
  *  isEqual(10, "10");
  *  //> false
  */
-function isEqual(a, b) {}
+ function isEqual(a, b) {
+  if (a === b || b === a){
+    return true
```

Comment:
I like the thoroughness of your thinking but both sides of your comparison on line 18 return the same thing so technically we do not need the repetition.